sources/CMakeLists.txt

```
@@ -16,11 +16,15 @@ target_sources(${PROJECT_NAME}
 target_include_directories(${PROJECT_NAME}
   PUBLIC
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+    $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
   PRIVATE
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src>
 )
 
-target_link_libraries(${PROJECT_NAME} PRIVATE Qt::Core Qt::Svg)
+target_link_libraries(${PROJECT_NAME}
+  PUBLIC
+    Qt::Core
+    Qt::Svg)
```

AFAICS the public headers use only `QString`, so Qt::Svg could remain `PRIVATE`.
```suggestion
target_link_libraries(${PROJECT_NAME}
  PUBLIC
    Qt::Core
  PRIVATE 
    Qt::Svg)
```